components/alert/src/Alert.js

```diff
@@ -0,0 +1,146 @@
+/* @jsx jsx */
+
+import React, { useState } from 'react';
+import { css, jsx, useTheme } from '@westpac/core';
+import Color from 'color';
+
+import { CloseIcon } from '../../icon/src'; //until icon package is published
+import { CSSTransition } from 'react-transition-group';
+
+// ==============================
+// Utils
+// ==============================
+
+// ==============================
+// Component
+// ==============================
+const styleAppearance = appearance => {
+	const theme = useTheme();
+	return css({
+		color: theme.colors[appearance],
+		display: 'flex',
+		alignItems: 'center',
+
+		'& > .icon': {
+			marginRight: '6px',
+			marginTop: '1px',
+			float: 'left',
+		},
+
+		'& > .icon-alert': {
+			marginRight: '4px',
+		},
+
+		a: {
+			color: 'inherit',
+		},
+	});
+};
+
+const styleBox = appearance => {
+	const theme = useTheme();
+	return css({
+		backgroundColor: Color('white')
+			.mix(Color(theme.colors[appearance]), 0.1)
+			.hex(),
+		display: 'block',
+		position: 'relative',
+		marginBottom: '21px',
+		zIndex: '1',
+		padding: '18px 34px 18px 18px',
+		borderTop: `1px solid ${Color('white')
+			.mix(Color(theme.colors[appearance]), 0.5)
+			.hex()}`,
+		borderBottom: `1px solid ${Color('white')
+			.mix(Color(theme.colors[appearance]), 0.5)
+			.hex()}`,
```

Comment:
much of these 'design' settings need to be set in the tokens rather than stored in the Alert component